frontend/include/chpl/framework/Context-detail.h

```diff
@@ -266,6 +266,7 @@ class QueryMapResultBase {
   // Whether or not errors from this query result have been shown to the
   // user (they may not have been if some query checked for errors).
   mutable bool emittedErrors = false;
+  mutable bool errorsPresentInSelfOrDependencies = false;
```

Comment:
Needs a comment describing how this field is used. It has a long and pretty descriptive name, but does "errors present" match "emittedErrors" for some dependency?